docs/metrics/learning-prometheus-http-server/README.md

```diff
@@ -0,0 +1,175 @@
+# Quick start on exporting metrics to Prometheus/Grafana
+
+- [Quick start on exporting metrics to Prometheus/Grafana](#quick-start-on-exporting-metrics-to-prometheusgrafana)
+  - [Prerequisite](#prerequisite)
+  - [Introduction](#introduction)
+  - [Get Prometheus](#get-prometheus)
+  - [Prometheus HTTP server](#prometheus-http-server)
+    - [Configuration](#configuration)
+    - [Start Prometheus](#start-prometheus)
+    - [Configure OpenTelemetry to Expose metrics to Prometheus Endpoint](#configure-opentelemetry-to-expose-metrics-to-prometheus-endpoint)
+    - [Check Results in Prometheus](#check-results-in-prometheus)
+    - [View/Query Results with Grafana](#viewquery-results-with-grafana)
+
+## Prerequisite
+
+It is highly recommended to go over the [getting-started](../getting-started/)
+project under the metrics document folder before following along this document.
+
+## Introduction
+
+- [What is Prometheus?](https://prometheus.io/docs/introduction/overview/)
+
+- [Grafana support for
+  Prometheus](https://prometheus.io/docs/visualization/grafana/#creating-a-prometheus-graph)
+
+## Get Prometheus
+
+Follow the [first steps]((https://prometheus.io/docs/introduction/first_steps/))
+in the prometheus official document.
+
+## Prometheus HTTP server
+
+### Configuration
+
+After downloading the [latest release](https://prometheus.io/download/), extract
+it to a local location that's easy to access. We will find the default
+prometheus configuration yaml file in the folder, named `prometheus.yml`.
+
+Replace all the content with:
+```
+global:
+  scrape_interval: 10s
+  scrape_timeout: 10s
+  evaluation_interval: 10s
+scrape_configs:
+- job_name: OpenTelemetryTest # assign a job name, as prometheus server can listen to multiple jobs
```

Comment:
```suggestion
- job_name: OpenTelemetryTest
```

Reply:
Or maybe something "MyOpenTelemetryDemo" (something that is self-explanatory)

Reply:
Updated.